README.md

```diff
@@ -107,6 +110,40 @@ This generates the following data to be sent.
 ```text
 producer.send(destination='/topic/my_route_key.v1', body='{"id": 1, "one": "Field One", "two": "Field Two"}')
 ```
+
+_**With multi destinations and serializers**_
+
+```python
+from django.db import models
+from django_outbox_pattern.decorators import publish
+
+
+@publish(destinations=['/topic/my_route_key_1', '/topic/my_route_key_2'],
```

Comment:
**thought**: I have a thought that keeping the link of the queue with the serializer by the array positions can generate a kind of doubt for some newer dev.
Maybe we can make a config dictionary where the link is between the object or we can make that clearer in the documentation. what do you think ?

```python
@publish(configs=[
    {
        "destination": "/topic/my_route_key_1",
        "serializer": "my_serializer_1"
    },
    {
        "destination": "/topic/my_route_key_2",
        "serializer": "my_serializer_2"
    }
])
```